<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDialog</class>
 <widget class="QDialog" name="SerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>串口号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>波特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>奇偶校验：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>停止位：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOK">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="serialportNameCombobox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baundRateComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="correctBitCombobox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stopBitComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dataBitsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
